dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/test-applications/ejbsubresource/src/io/openliberty/jaxrs/ejbsubresource/EJBSubResourceClientTestServlet.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.jaxrs.ejbsubresource;
+
+
+import static org.junit.Assert.assertEquals;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+
+@SuppressWarnings("serial")
+@WebServlet(urlPatterns = "/EJBSubResourceClientTestServlet")
+public class EJBSubResourceClientTestServlet extends FATServlet {
+
+    private static final String URI_CONTEXT_ROOT = "http://localhost:" + Integer.getInteger("bvt.prop.HTTP_default") + "/ejbsubresource/";
+
+    private Client client;
+
+    @Override
+    public void before() throws ServletException {
+        client = ClientBuilder.newClient();
+    }
+
+    @Override
+    public void after() {
+        client.close();
+    }
+
+    @Test
+    public void testIsub() {
+        Response response = client.target(URI_CONTEXT_ROOT)
+                        .path("root/sub")
+                        .request(MediaType.TEXT_PLAIN_TYPE)
+                        .get();
+        assertEquals(200, response.getStatus());
+        assertEquals("http://localhost:8010/ejbsubresource/root/sub", response.readEntity(String.class));
```

Comment:
Is this guaranteed to be 8010?  Should it be using the URI_CONTEXT_ROOT to know the beginning of the URI?

Reply:
it should, good catch